sd-2019-stack-overflow-frontend/src/view/QuestionOperations.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const QuestionOperations = ( {onAskQuestion, titleFilter, tagFilter, onChangeTitleFilter, onChangeTagFilter} ) => (
+    <section className="container" style={{padding: '10px 0px 10px 0px'}}>
```

Comment:
Usually it is considered bad practice to use inline styles. Think about creating your own CSS class instead.